src/lib/stories.js

```diff
@@ -187,6 +246,62 @@ const printStory = (program) => { return (story) => {
     return story;
 };};
 
+//TODO: Add workspace name in URL to avoid extra redirect.
+const storyURL = (story) => `https://app.clubhouse.io/story/${story.id}`;
+
+const printDetailedStory = (story) => {
+    const labels = story.labels.map(l => {
+        return chalk.bold(`#${l.id}`) + ` ${l.name}`;
+    });
+    const owners = story.owners.map(o => {
+        return `${o.profile.name} (` + chalk.bold(`${o.profile.mention_name}` + ')');
+    });
+
+    log(chalk.blue.bold(`#${story.id}`) + chalk.blue(` ${story.name}`));
+    log(chalk.bold('Desc:') + `     ${formatLong(story.description || '_')}`);
+    log(chalk.bold('Owners:') + `   ${owners.join(', ') || '_'}`);
+    log(chalk.bold('Type:') + `     ${story.story_type}/${story.estimate || '_'}`);
+    log(chalk.bold('Label:') + `    ${labels.join(', ') || '_'}`);
+    log(chalk.bold('Project:') + chalk.bold(`  #${story.project_id} `) + story.project.name);
+    if (story.epic) {
+        log(chalk.bold('Epic:') + chalk.bold(`     #${story.epic_id} `) + story.epic.name);
+    } else {
+        log(chalk.bold('Epic:') + '     _');
+    }
+    log(chalk.bold('State:') + chalk.bold(`    #${story.workflow_state_id} `) + story.state.name);
+    log(chalk.bold('Created:') + `  ${story.created_at}`);
+    if (story.created_at !== story.updated_at) {
+        log(chalk.bold('Updated:') + `  ${story.updated_at}`);
+    }
+    log(chalk.bold('URL:') + `      ${storyURL(story)}`);
+    if (story.archived) {
+        log(chalk.bold('Archived: ') + chalk.bold(story.archived));
+    }
+    if (story.completed) {
+        log(chalk.bold('Completed: ') + chalk.bold(story.completed_at));
+    }
+    story.tasks.map(c => {
+        log(chalk.bold('Task:     ') + (c.complete ? '[X]' : '[ ]')
+            + ' ' + formatLong(c.description));
+        return c;
+    });
+    story.comments.map(c => {
+        const author = members.filter(m => m.id === c.author_id)[0]
```

Comment:
`club story 4532` currently fails here with `ReferenceError: members is not defined`

Reply:
Good catch, all my test stories had no comments. X|

Did you find any more bugs?

Reply:
Not yet! Thanks.